compiler/luci/pass/src/ForwardReshapeToUnaryOpPass.cpp

```diff
@@ -180,6 +258,100 @@ class ForwardReshape final : public luci::CircleNodeMutableVisitor<bool>
 
     return forward_reshape(reshape, node);
   }
+
+  bool visit(luci::CircleDiv *node)
+  {
+    luci::CircleReshape *reshape = nullptr;
+    luci::CircleConst *const_value = nullptr;
```

Comment:
This forward the reshape through the div (one of the input is scalar const value)

Reply:
Can you use `NodeFiller.h`? That will make code much easy to read.

Reply:
Done